openmdao/docs/usr-guide/tutorials/recording.rst

```diff
@@ -96,12 +94,34 @@ Actually, OpenMDAO's `SqliteRecorder` makes use of the
 `sqlitedict module <https://pypi.python.org/pypi/sqlitedict>`_ because it has a
 simple, Pythonic dict-like interface to Python’s sqlite3 database.
 
-We then attach the recorder to the driver using `driver.add_recorder`.
-Depending on your needs, you are able to attach more recorders by using
+We then add the recorder to the driver using `driver.add_recorder`.
+Depending on your needs, you are able to add more recorders by using
 additional `driver.add_recorder` calls. Solvers also have an `add_recorder`
 method that is invoked the same way. This allows you to record the evolution
 of variables at lower levels.
 
+While it might not be an issue, it is good practice to close the 
+recorder explicitly before the program terminates.
+For this tutorial with one recorder added to the driver, this is simply done with:
+
+::
+
+    driver.recorders[0].close()
+
+If your model has recorders added to both drivers and solvers,
+a way to make sure all recorders are closed is to use code like this:
+
+::
+
+    for recorder in top.driver.recorders:
```

Comment:
OK.  Whoever works on that story will just have to remember to update this part of the docs.